scripts/python/packages/parthenon_tools/parthenon_tools/phdf.py

```diff
@@ -648,11 +648,11 @@ def Get(self, variable, flatten=True, interior=False, average_to_cell_centers=Tr
         if flatten:
             nblocks = vShape[0]
             if self.varTopology[variable] == "None":
-                remaining_size = np.product(vShape[1:])
+                remaining_size = np.prod(vShape[1:])
```

Comment:
glad to see that the numpy gods thought this was a necessary change.

Reply:
actually, does `prod` work for older versions of numpy or will this break the test suite when not using the newer numpy?

Reply:
Looks like the change should be backwards compatible: https://stackoverflow.com/questions/49863633/numpy-product-vs-numpy-prod-vs-ndarray-prod.